FinniversKit/Sources/DNA/Color/UIColor+FinniversKit.swift

```diff
@@ -137,6 +137,10 @@ import UIKit
         return UIColor(r: 255, g: 255, b: 255)
     }
 
+    public class var darkMilk: UIColor {
```

Comment:
It's basically the name of that hex value defined in `.bgPrimary` 😉 

Some SwiftUI views can't automatically adapt to dark mode so we need to listen to the colour scheme and apply the appropriate colours.

IMHO this 

`.background(colorScheme == .dark ? Color(.darkMilk) : Color(.milk))`

just reads better, than this

`.background(colorScheme == .dark ? Color(.bgPrimary) : Color(.bgPrimary))`.

Reply:
In which case doesn't SwiftUI adapt to dark mode? 🤔

Reply:
I couldn't reproduce this in the framework and it worked fine in testing on iOS 14, 15 and 16, so I changed it back to dynamic colors. I will leave darkMilk in FinniverseKit though since all available colors should be exposed.